zebra/zebra_ns.h

```diff
@@ -87,6 +87,40 @@ struct zebra_ns {
 	struct ns *ns;
 };
 
+/* Key netlink info from zebra ns, passed from the zebra main context
+ * to the dataplane/kernel context (which might be in a different pthread).
+ */
+struct zebra_ns_info {
```

Comment:
I think this is the wrong abstraction.  Dataplanes own this data not namespaces, in this case the netlink dataplane owns this.  This is a implementation detail of the netlink dataplane.  Why do namespaces care?

It sure seems like we should pull out the netlink data and place ownership into the netlink dataplane.